File: messages/src/main/java/com/thundashop/core/sendregning/NewInvoice.java
package com.thundashop.core.sendregning;

import java.util.List;

public class NewInvoice {
    String orderNumber;
    String InvoiceDate;
    String dueDate;
    String orderDate;
    String ourReference;
    String yourReference;
    String invoiceText;
    String deliveryDate;
    NewInvoiceAddress deliveryAddress;
    NewInvoiceRecipient recipient;
    NewInvoiceShipment shipment = new NewInvoiceShipment();
    List<NewInvoiceItem> items;
    
}

File: html5apps/terminal/events/core/sendregning/NewInvoice.php
<?php
class core_sendregning_NewInvoice {
	/** @var String */
	public $orderNumber;

	/** @var String */
	public $InvoiceDate;

	/** @var String */
	public $dueDate;

	/** @var String */
	public $orderDate;

	/** @var String */
	public $ourReference;

	/** @var String */
	public $yourReference;

	/** @var String */
	public $invoiceText;

	/** @var String */
	public $deliveryDate;

	/** @var core_sendregning_NewInvoiceAddress */
	public $deliveryAddress;

	/** @var core_sendregning_NewInvoiceRecipient */
	public $recipient;

	/** @var core_sendregning_NewInvoiceShipment */
	public $shipment;

	/** @var core_sendregning_NewInvoiceItem[] */
	public $items;

}
?>